ls1intum/Artemis
# Feature/programming exercises/enable c jenkins

### Checklist

### Motivation and Context


Generating a C programming exercise in Jenkins is not possible because the pipeline configuration using the plugins which will be deprecated in the future.

### Description

The configuration was ported into Jenkins Pipeline by @sleiss. I made minor changes and re-activated the programming language in Jenkins

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> Exercises
3. Click on Generate new Programming Exercise and select C as the programming language
4. Participate in the exercise by opening the code editor
5. Press submit and simply make sure that we get some feedback (should be near identical to the feedback from TS1)
